README.md

```diff
@@ -35,14 +35,16 @@ Then, install Python Cozy Management :
 * **get_oom_scores**: get the Out Of Memory score for each application;
 * **rebuild_app <app>**: reinstall every npm dependencies of an application;
 * **rebuild_all_apps**: reinstall every npm dependencies of all applications (useful when upgrading Node version);
-* **migrate_2_node4**: 
-* **install_requirements**: 
+* **migrate_2_node4**:
+* **install_requirements**:
 * **install_cozy**: install the Cozy server;
 * **wait_couchdb**: wait until couchdb has started;
 * **wait_cozy_stack**: wait until full Cozy stack has started;
 * **emulate_smtp [--bind <ip>] [--port <port>]** starts a fake SMTP server on port 25 for debugging purpose;
 * **backup**: create a timestamped backup of Cozy configuration and data into `/var/lib/cozy/backups`;
 * **restore <backup_filename>**: restore a backup;
+* **install_weboob**: install weboob from devel repository
+* **update_weboob_modules**: update weboob modules
```

Comment:
For what it's worth, Kresus (at least) takes care of the modules you need for your bank accounts, stores them within the kresus/ directory and updates them on your behalf. So this is not strictly needed to have this command, per se; other users of weboob in tree might want to do things differently.

That being said, it'd be nice to be able to update weboob-core, which is installed with `install_weboob` but never updated. If you trip in an unstable version of the `devel` tree, what you want to do is update the weboob core installation itself, not the modules.